tests/test_tokenization_common.py

```diff
@@ -356,7 +357,7 @@ def test_sentencepiece_tokenize_and_convert_tokens_to_string(self):
             return
 
         tokenizer = self.get_tokenizer()
-        text = "This is text to test the tokenizer."
+        text = "This is a test"
```

Comment:
Is there a particular reason why you changed this example? :slightly_smiling_face:

Reply:
Yes, it's because of the test of `MarianTokenizer` uses a very specific and small vocabulary, as shown in the snippet, so I thought it wouldn't make much of a difference to the other tokenizers if I made that change. Without the change, `MarianTokenizer` fails the test.

https://github.com/huggingface/transformers/blob/35ecf99cc4e1046e5908f5933a4548e43a0f4f1d/tests/marian/test_tokenization_marian.py#L57

Reply:
I am surprised because I tested your test by changing only this value and I do not encounter any errors for MarianTokenizer. Could you please re-test and share the error you had? :pray:

Reply:
Well, that's embarrassing. I know `MarianTokenizer` gave me an error before, but now it no longer does. I reverted the `text` variable to be the original string.